tool/src/js/wasm.mjs

```diff
@@ -5,13 +5,22 @@ let wasm;
 
 const imports = {
   env: {
-    log_js(ptr, len) {
+    console_debug_js(ptr, len) {
```

Comment:
Yeah the problem is someone using icu4x would get an automatic update of diplomat-runtime. This may be an acceptable thing to break for now since JS is still very iffy (cc @sffc?) and the logging isn't proprely hooked up.

Also can we scope it as `diplomat_console_...`?

Reply:
Yeah this is breaking, but releasing major diplomat versions is not a lot of work.

Reply:
sgtm

Reply:
I'd rather fix JS than go out of our way to make it backwards compatible with clients I don't think we have right now.